python/ray/serve/experimental/gradio_visualize_graph.py

```diff
@@ -39,6 +41,73 @@ def lazy_import_gradio():
     return _gradio
 
 
+def get_block(node, **kwargs):
+    gr = lazy_import_gradio()
+
+    return_type = node.get_return_type()
+    node_name = _DAGNodeNameGenerator().get_node_name(node)
+    if return_type is None:
+        logger.warning(
+            f"Return type for {node_name} node was not provided. "
+            "Defaulting to gr.Textbox."
+        )
+        return gr.Textbox(**kwargs)
+
+    if return_type == "int":
+        return gr.Number(precision=0, **kwargs)
+    elif return_type == "float":
+        return gr.Number(**kwargs)
+    elif return_type == "str":
+        return gr.Textbox(**kwargs)
+    elif return_type == "bool":
+        return gr.Checkbox(**kwargs)
+    elif return_type == "pandas.core.frame.DataFrame":
+        return gr.Dataframe(**kwargs)
+    elif (
+        return_type == "list"
+        or return_type == "dict"
+        or return_type == "typing.List"
+        or return_type == "typing.Dict"
+    ):
+        return gr.JSON(**kwargs)
+    else:
+        from PIL import ImageFile
```

Comment:
If users provide an invalid type, and they don't have `PIL` installed, they're going to get a strange error about `PIL` not being installed instead of an error saying their type is invalid. Could we raise an invalid type error in that case?